framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -203,50 +204,122 @@ private ATypeElement getParameterType(
     /**
      * Get the annotations for the receiver type.
      *
-     * @param methodAnnos the method or constructor annotations
+     * @param methodElt the method or constructor
      * @param paramATM the receiver type
      * @param atypeFactory the type factory
      * @return the annotations for the receiver type
      */
     @SuppressWarnings("UnusedVariable")
     private ATypeElement getReceiverType(
-            AMethod methodAnnos, AnnotatedTypeMirror paramATM, AnnotatedTypeFactory atypeFactory) {
+            ExecutableElement methodElt,
+            AnnotatedTypeMirror paramATM,
+            AnnotatedTypeFactory atypeFactory) {
+        AMethod methodAnnos = getMethodAnnos(methodElt);
         return methodAnnos.receiver.type;
     }
 
     /**
      * Get the annotations for the return type.
      *
-     * @param methodAnnos the method or constructor annotations
+     * @param methodElt the method or constructor
      * @param atm the return type
      * @param atypeFactory the type factory
      * @return the annotations for the return type
      */
     @SuppressWarnings("UnusedVariable")
     private ATypeElement getReturnType(
-            AMethod methodAnnos, AnnotatedTypeMirror atm, AnnotatedTypeFactory atypeFactory) {
+            ExecutableElement methodElt,
+            AnnotatedTypeMirror atm,
+            AnnotatedTypeFactory atypeFactory) {
+        AMethod methodAnnos = getMethodAnnos(methodElt);
         return methodAnnos.returnType;
     }
 
     /**
      * Get the annotations for a field type.
      *
-     * @param classAnnos the class annotations
+     * @param element the element for the field
      * @param fieldName the simple field name
      * @param lhsATM the field type
      * @param atypeFactory the annotated type factory
      * @return the annotations for a field type
      */
-    public ATypeElement getFieldType(
-            AClass classAnnos,
+    @SuppressWarnings("UnusedVariable")
```

Comment:
Is this still needed?

Reply:
`atypeFactory` is not used.  (But it will be used in a future refactor.)